libs/runtime/ebpf_hash_table.c

```diff
@@ -593,6 +595,14 @@ ebpf_hash_table_create(_Out_ ebpf_hash_table_t** hash_table, _In_ const ebpf_has
     void* (*allocate)(size_t size) = options->allocate ? options->allocate : ebpf_epoch_allocate;
     void (*free)(void* memory) = options->free ? options->free : ebpf_epoch_free;
 
+    // Increase bucket_count to next power of 2.
```

Comment:
The options has a field called bucket_count which now seems to be incorrect.  Should it be renamed to "minimum_bucket_count"?

Reply:
Done.